examples/mugen/retrieval/video_clip.py

```diff
@@ -8,16 +8,19 @@
 from typing import Any, Dict, Optional
 
 import torch
-from examples.mugen.retrieval.s3d import S3D
+
 from torch import nn
 
 from torchmultimodal.models.clip.model import CLIP
 from torchmultimodal.utils.common import load_module_from_url
+from torchvision.models.feature_extraction import create_feature_extractor
+
+from torchvision.models.video import S3D
 from transformers import DistilBertConfig, DistilBertModel
 
 
 PRETRAINED_S3D_KINETICS400_URL = (
-    "https://pytorch.s3.amazonaws.com/models/multimodal/mugen/S3D_kinetics400.pt"
+    "https://pytorch.s3.amazonaws.com/models/multimodal/mugen/S3D_kinetics400_v1.pth"
```

Comment:
I think it might be a good idea to get some hash from the file as part of the name (this is useful for versioning as well). In torchvision we get the hash in [here](https://github.com/pytorch/vision/blob/main/references/classification/utils.py#L387) (note that we only take the first 8 characters of the hash digest).

Reply:
Thanks for the suggestion. I have re-generated the filename using hexdigest. Will probably include TorchVision's util in my next PR as well.